and-bible/app/src/main/res/values/strings.xml

```diff
@@ -128,6 +128,9 @@ TODO: error and download_document_confirm_prefix need to be parameterized rather
     <string name="prefs_split_mode_summary">How to split multiple screens on the window layout?</string>
     <string name="prefs_split_mode_title">Split mode</string>
 
+    <string name="prefs_request_sdcard_permission_title">Request permission to read external storage</string>
```

Comment:
This setting is specific to manually installed modules and should be more specific e.g. 'Manually installed documents exist'
and for the summary:
Enable use of manually installed documents from the jsword folder on the SD card.'

Reply:
Changed this as suggested